tensorcircuit/applications/__init__.py

```diff
@@ -3,3 +3,5 @@
 the code inside is subject to change, be caution to use.
 Most of the useful code is and will be refactored and copied to other parts of tensorcircuit.
 """
+
+from . import optimization
```

Comment:
I dont suggest to explicitly import any submodule in applications folder, but you can still use them later anyway